app/views/internal/mods/index.html.erb

```diff
@@ -49,7 +49,7 @@
       <% if params[:state] == "potential" %>
         <td>
           <%= form_with model: [:internal, mod], url: internal_mod_path(mod.id), method: :patch, local: true do |f| %>
-            <%= f.submit "Make Trusted Mod", class: "btn btn-light" %>
+            <%= f.submit "Make Trusted Mod", class: "btn btn-light js-add-to-mod-channel" %>
```

Comment:
yes, I so agree with @msarit about making classes used by JS explicit with the prefix, it's a game changer to be able to understand where JS behavior comes from. Not always easy and not always necessary but so helpful